polls/tests/test_voting.py

```diff
@@ -75,7 +75,7 @@ def test_voting_once_only(self):
         self.assertContains(response, 'is this a test')
         response = client.post(reverse('molo.polls:vote',
                                kwargs={'question_id': question.id}))
-        self.assertContains(response, "You didn&#39;t select a choice")
+        self.assertContains(response, "field is required")
```

Comment:
@miltontony should I keep you didn't select a choice or display the error using form.choice.errors?